templates/cisco_ios_show_cdp_neighbors.template

```diff
@@ -1,14 +1,13 @@
 Value Required NEIGHBOR (\S+)
-Value LOCAL_INTERFACE (\S+\s\S+)
-Value CAPABILITY ((?:\w(?:\s(?!\s))?){0,})
-Value PLATFORM (\S{2,}\s\S{2,}|\S+)
-Value NEIGHBOR_INTERFACE (\S+\s\S+)
+Value LOCAL_INTERFACE (\S+(?:\s\S+)?)
+Value CAPABILITY ((?:\w\s*?)*?)
+Value PLATFORM (\S{2,}\s\S+|\S{2,})
+Value NEIGHBOR_INTERFACE ([a-zA-Z]\S+(?:\s\S+)?)
 
 Start
   ^Device.*ID -> CDP
 
 CDP
   ^${NEIGHBOR}$$
-  ^\s+${LOCAL_INTERFACE}\s+\d+\s+${CAPABILITY}\s+${PLATFORM}\s+${NEIGHBOR_INTERFACE} -> Record
-  ^${NEIGHBOR}\s+${LOCAL_INTERFACE}\s+\d+\s+${CAPABILITY}\s+${PLATFORM}\s+${NEIGHBOR_INTERFACE} -> Record
-
+  ^\s+${LOCAL_INTERFACE}\s+\d+\s+${CAPABILITY}\s*?${PLATFORM}\s+${NEIGHBOR_INTERFACE} -> Record
```

Comment:
Can you change from **\s+** to **\s*** for L12? If you look at the line with router.domain.local, it isn't captured properly.

After that, I think everything looks good to go.